design/CERTIFICATE_SIGNING.md

```diff
@@ -1,32 +1,100 @@
+Digital certificates are a useful mechanism for authentication, data
+integrity, and data confidentiality. Certificates offer many advantages
+over other options such as passwords, API keys, or public/private key
+pairs because they are revokable, have built-in mechanisms for validity
+checking, may include attributes describing the subject they identify,
+and most importantly, they can be signed by a trusted authority.
 
-- A running Conjur instance
-- A policy to configure the CA and hosts to request certificates
-- A certificate chain and private key for the issuer CA
+The use of trusted authorities, and a public key infrastructure (PKI)
+make digital certificates (and certificate signing) a cornerstone of
+flexible and scalable access control architecture.
 
-### Policy for Conjur CAs
+Managing private key material for trusted authorities, and controlling
+and auditing the use of signatures can be complicated and risky.
+Certificate signing in Conjur is a capability intended to leverage the
+existing role and strengths of Conjur in an authorization architecture
+to promote the safe and effective use digital certificates.
 
-The policy for configuring a Conjur Certificate authority begins with
-a `webservice` entity with a well-known id of the form
-`conjur/<ca-service-id>/ca`. The CA is configured using well-known
-annotations. A minimal CA policy might look like:
+## Use Cases
+
+Conjur certificate authorities may be used to sign digital certificates
+addressing 4 use cases:
+
+1. **TLS Host Certificates**
+2. **TLS Client Certificates**
+3. **SSH Host Certificates**
+4. **SSH Client Certificates**
+
+## Getting Started with Conjur Certificate Authorities
+
+All Conjur Certificate Authorities (CA's) begin with a policy
+definition and a private key.
+
+### Create a Private Key
+In the alpha version, Conjur certificate authorities only support
+RSA private keys in PEM format. A valid private key may be created
+either using `openssl` or `ssh-keygen`:
+```sh
+openssl genrsa -aes256 -out private_key_file 4096 
+```
+```
+ssh-keygen -t rsa -m pem -f private_key_file
+```
+
+### (x.509) Create a Signing Certificate
+
+Conjur CAs in the alpha version requre the issuer certificate to be
+generated outside of Conjur. A good walk through of creating a either
+a self-signed root CA certificate or an intermediate CA certificate
+that may be used in conjur is available
+[here](https://jamielinux.com/docs/openssl-certificate-authority/index.html).
+
+### Define a Certificate Authority in Conjur Policy
+
+A Conjur Certificate Authority is creating in Conjur by defining
+a `webservice` entity with a well-known identifier of the form
 ```
+conjur/ca/<service_id>
+```
+where `<service_id>` is any value you choose.
+
+The certificate authority is configured with annotations on the
+`webservice`. The available configuration options are:
+
+- `ca/private-key` (Required)
+  <br/> Id of conjur variable that contains the PEM encoded CA private key.
```

Comment:
I believe you can remove the `<br/>` and replace with indentation of 4 spaces to achieve a line break.